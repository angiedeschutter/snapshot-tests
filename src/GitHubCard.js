import Card from 'react-bootstrap/Card';

function GitHubCard() {
  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src='https://placekitten.com/400/400' />
      <Card.Body>
        <Card.Title>User Name</Card.Title>
        <Card.Text>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis turpis lacus, tempor nec laoreet at, ultrices nec justo.
        </Card.Text>
      </Card.Body>
    </Card>
  );
}

export default GitHubCard;