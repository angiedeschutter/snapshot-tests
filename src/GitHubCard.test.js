import renderer from 'react-test-renderer'

import GitHubCard from "./GitHubCard"

test('it renders a snapshot of card', ()=>{
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
  })