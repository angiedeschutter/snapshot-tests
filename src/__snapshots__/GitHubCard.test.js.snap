// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a snapshot of card 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="https://placekitten.com/400/400"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      User Name
    </div>
    <p
      className="card-text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis turpis lacus, tempor nec laoreet at, ultrices nec justo.
    </p>
  </div>
</div>
`;
